src/enums/filingCodes.ts

```diff
@@ -3,5 +3,6 @@ export enum FilingCodes {
   INCORPORATION_CP = 'OTINC',
   INCORPORATION_BC = 'BCINC',
   CORRECTION = 'CRCTN',
-  ALTERATION = 'ALTER'
+  ALTERATION = 'ALTER',
+  CHANGE_OF_REGISTRATION = 'FMCHANGE'
 }
```

Comment:
This is a shared enum, right?

Could you please add the missing code? Eg,
https://github.com/bcgov/business-create-ui/blob/main/src/enums/filingCodes.ts

Reply:
For sure